instances/latest/commonCoordinateSpaceVersions/MBISC/MBISC_v2012.jsonld

```diff
@@ -0,0 +1,125 @@
+        "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+    },
+    "axesOrigin": [
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        },
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        },
+        {
+            "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+            "typeOfUncertainty": null,
+            "uncertainty": null,
+            "unit": {
+                "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/millimeter"
+            },
+            "value": 0
+        }
+    ],
+    "releaseDate": "2011-10-27",
+    "copyright": {
+        "@type": "https://openminds.om-i.org/types/Copyright",
+        "holder": {
+            "@type": "https://openminds.om-i.org/types/Organization",
+            "fullName": "Elsevier Inc."
+        },
+        "year": "2012"
+    },
+    "usedSpecimen": [
+        {
+            "@type": "https://openminds.om-i.org/types/Subject",
+            "species": {
+                "@id": "https://openminds.om-i.org/instances/species/callithrixJacchus"
+            },
+            "studiedState": [
+                {
+                    "@type": "https://openminds.om-i.org/types/SubjectState",
+                    "ageCategory": {
+                        "@id": "https://openminds.om-i.org/instances/ageCategory/youngAdult"
+                    },
+                    "age": {
+                        "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+                        "typeOfUncertainty": null,
+                        "uncertainty": null,
+                        "value": "3.16",
+                        "unit": {
+                            "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/year"
+                        }
+                    },
+                    "weight": {
+                        "@type": "https://openminds.om-i.org/types/QuantitativeValue",
+                        "typeOfUncertainty": null,
+                        "uncertainty": null,
+                        "value": "500",
+                        "unit": {
+                            "@id": "https://openminds.om-i.org/instances/unitOfMeasurement/gram"
+                        }
+                    }
+                }
+            ],
+            "biologicalSex": {
+                "@id": "https://openminds.om-i.org/instances/biologicalSex/female"
+            }
+        }
+    ],
```

Comment:
specimen cannot be nested. for now please just delete the subject information for the common coordinate space version completely. we can add them later once we decide within openMINDS how to handle those.